process/smartContract/process.go

```diff
@@ -444,14 +457,58 @@ func (sc *scProcessor) executeSmartContractCall(
 	return vmOutput, nil
 }
 
+func (sc *scProcessor) isSCROnlyInformational(scr data.TransactionHandler) bool {
+	if scr.GetValue().Cmp(zero) > 0 {
+		return false
+	}
+
+	function, _, err := sc.argsParser.ParseCallData(string(scr.GetData()))
+	if err != nil {
+		return true
```

Comment:
is there any situation when a ParseCallData can fail and the SCR will be treated the new way?

Reply:
parseData will fail if the scr.Data is not a function cal, not a smart contract execution and it is treated as move balance only. But this scr does not have eGLD so it is informational only.